### PR TITLE
Fix astropy deprecations in tests

### PR DESCRIPTION
**Pull request by astrofrog**
There are a few tests that use deprecated behavior - I think it would be good to include in 3.0

### CODE REVIEW

File: astropy/io/fits/tests/test_image.py [hunk 1]
```diff
@@ -1208,8 +1208,8 @@ def test_comp_image_hcompression_1_invalid_data(self):
 
         pytest.raises(ValueError, fits.CompImageHDU,
                       np.zeros((2, 10, 10), dtype=np.float32), name='SCI',
-                      compressionType='HCOMPRESS_1', quantizeLevel=16,
-                      tileSize=[2, 10, 10])
+                      compression_type='HCOMPRESS_1', quantize_level=16,
+                      tile_size=[2, 10, 10])
```
Comment (saimn) [inline astropy/io/fits/tests/test_image.py]: The `ignore_warnings` decorator can probably be removed then ?

File: astropy/io/fits/tests/test_image.py [hunk 2]
```diff
@@ -1223,8 +1223,8 @@ def test_comp_image_hcompress_image_stack(self):
 
         cube = np.arange(300, dtype=np.float32).reshape(3, 10, 10)
         hdu = fits.CompImageHDU(data=cube, name='SCI',
-                                compressionType='HCOMPRESS_1',
-                                quantizeLevel=16, tileSize=[5, 5, 1])
+                                compression_type='HCOMPRESS_1',
+                                quantize_level=16, tile_size=[5, 5, 1])
```
Comment (saimn) [inline astropy/io/fits/tests/test_image.py]: Same here.

File: astropy/io/fits/tests/test_image.py (hunk 1 above)
Comment (astrofrog) [reply astropy/io/fits/tests/test_image.py]: Good catch, thanks!

File: astropy/io/fits/tests/test_image.py (hunk 2 above)
Comment (astrofrog) [reply astropy/io/fits/tests/test_image.py]: Done

**Pull request merged**